src/dotnet-svcutil/lib/src/Metadata/ServiceDescriptor.cs

```diff
@@ -107,7 +107,7 @@ public async Task ImportMetadataAsync(Action<WsdlImporter> onWsdlImporterCreated
                 }
                 else
                 {
-                    tfn = "net6.0";
+                    tfn = Environment.Version.Major >= 8 ? "net8.0" : "net6.0";                        
```

Comment:
Is this only relying on .NET 8.0?  If so, throw some kind of exception?  Is this protected somewhere else?  If so, at least add a comment.